test/integration/options/watch.spec.js

```diff
@@ -49,6 +138,25 @@ describe('--watch', function() {
       });
     });
 
+    it('picks up new test files when they are added', function() {
```

Comment:
This test is identical to the one above.

Reply:
Done